### PR TITLE
split lxgwnewcleargothic and lxgwfasmartgothic font

### PR DESCRIPTION
**Pull request by stdmnpkg**
Topic Description
-----------------

- lxgwnewcleargothic-font: transitionalize => lxgwneoxihei-font, lxgwfasmartgothic-font
- lxgwfasmartgothic-font: new, 1.200, split from lxgwnewcleargothic-font
- lxgwneoxihei-font: update to 1.200, renamed from lxgwnewcleargothic-font

Package(s) Affected
-------------------

Security Update?
----------------

No

Build Order
-----------

```
#buildit lxgwfasmartgothic-font lxgwnewcleargothic-font  lxgwneoxihei-font
```

Test Build(s) Done
------------------

**Primary Architectures**

- [ ] AMD64 `amd64`
- [ ] AArch64 `arm64`
- [ ] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Review by eatradish — DISMISSED** [review]
Should create a transitional package and recommend NeoXiHei and Fasmartgothic?

File: desktop-fonts/lxgwfasmartgothic-font/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,9 @@
+PKGNAME="lxgwfasmartgothic-font"
+PKGDES="A Sans Serif font derived from 03 Smartfont UI"
+PKGDEP="fontconfig"
+PKGSEC="fonts"
+
+ABHOST="noarch"
+
+PKGBREAK="lxgwnewcleargothic<=1.124"
```
Comment (MingcongBai) [inline desktop-fonts/lxgwfasmartgothic-font/autobuild/defines]: Wrong package name reference:

```
dpkg: 处理归档 /tmp/apt-dpkg-install-AXSBep/08-lxgwneoxihei-font_1.200~pre20240825T192121Z_all.deb (--unpack)时出错：
 正试图覆盖 /usr/share/fonts/TTF/LxgwNeoXiHei-Book.ttf，它同时被包含于软件包 lxgwnewcleargothic-font 1.124
```

File: desktop-fonts/lxgwneoxihei-font/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,9 @@
+PKGNAME="lxgwneoxihei-font"
+PKGDES="A Chinese sans-serif font derived from IPAex Gothic"
+PKGDEP="fontconfig"
+PKGSEC="fonts"
+
+ABHOST="noarch"
+
+PKGBREAK="lxgwnewcleargothic<=1.124"
```
Comment (MingcongBai) [inline desktop-fonts/lxgwneoxihei-font/autobuild/defines]: Ditto.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]
Tested good on LoongArch.

**Pull request merged**